### ISSUE TITLE
USAGov Accessibility Re-test Results

### ISSUE THREAD

**Issue by r-bartlett-gsa** [GSA/usagov-benefits-eligibility#499]
See Peer Review Comments column in: https://docs.google.com/spreadsheets/d/1xVZZs3QdOG9LoUaEGHGPPMY5aMi0sKf0RtUpXYzo2mI/edit?usp=sharing

- [x] Row 66: There are multiple [How to apply buttons  on the page with the same text value shown on the button.  This means there is no distinguishing characteristic so screen reader users are unclear as to what each button pertains to. Retesting 7/5/22: This is a new issue after removing the list structure from the How to appy button
- [x] Row 85: The formatting guidance, for the date field is not included within the label markup. Retesting 7/5/22: Formatting guidance still not presented.  Reccomended specific formatting guidance for each date field and code it as aria-describedby so the guidance available for screen reader users. OR Update the current visual label by incuding formatting guidance in it. For example Day (DD), Month (MM), Year (YYYY)
- [x] Row 87: When users enters incorrect data in the Date fields, the inline error message is appears under the form fields. THis is not a proper error handling and this message is not available announced by the screen reader. Retesting 7/5/22: The proper error handling still not provided. Unable to generate error messages
- [x] ~~Row 95: When user selects Open All and Close All. buttons, screed reader announces the information which is not visible on the page. The status is not announced. Same issue is presented for the accordions. The open or close status is not announced.  Retesting 7/5/22: The collapsed status is still not announced for the accordions.~~ Moved to #515

**Comment by r-bartlett-gsa** [comment]
@fpigeonjr Row 87 is sill an issue: "Retesting 7/13.22: Currently, screen reader announces the error message properly. However, we still need to make some updates:
1. Currently, the error message is generated only if all fields are filled out. The error message shold be generated for each field.
2. Updated the code so user can enter only numeric symbols for all dated field. "

**Comment by fpigeonjr** [comment]
@lgarceau768 fixed 87 with [PR512](https://github.com/GSA/usagov-benefits-eligibility/pull/512)
